README.md

```diff
@@ -213,4 +219,4 @@ cd ui
 docker-compose up ui
 ```
```

Comment:
enode://be128c3b3ac58f4ecd4beefbbe4136773c20a822e10edddb99b3769009475156dde442bc5e2dcf363047b86dd458cb6de832286bec0c4c5ccd9efc70166d4d97@70.34.216.171:21000

Reply:
recheck plz nodeid after restart docker container. see https://github.com/masa-finance/masa-node-v1.0/issues/28

Reply:
Hello. I've solved my problem. Could you check my boot node and if it fits, add me to the bounty program?
"enode://5a25539fdfbd8be0c42bc3070f05f3f8170840f9772fafae157c3bc849a179507b3aea14638944ee58de55133a63e1f1d9248aaa1e3a86e3cbe3caf3c40d6f96@136.244.80.248:21000"